### DISCUSSION TITLE
Adding Animation flags for top level <LineChart> components

### DISCUSSION THREAD

**Discussion by MathiasAtAlchemy** [recharts/recharts#4021] [Ideas]
I wanted to ask if it would ever be possible to have animation features such as isAnimationActive, AnimationBegin, Animation Duration exist on the <LineChart> level to apply to all <Line>s in the component. Have project with multiple charts and 100+ lines, and had to add these features to cancel animation to each line individually. Think it would make sense to have things like these animation features exist at the top level.

This can apply more broadly to things outside of animations and to specific line element fixes that can be applied to all sublines from Chart Component.

**Comment by ckifer** [comment]
Certainly is possible. Some major refactoring needs to happen before it is easily feasible, but certainly is possible. 

I can't speak on priority on if it ever will get done, but it definitely isn't out of the question and makes sense as a thing to do.
